Improvement/ysmod 637 dokumentmottak: konvertere aktoerid til fnr

olsnerik opened the pull request:
Skal ikke lagre aktørId i vår løsning. Må over til å lagre fødselsnummer. Oppdatert dokumentmottaket til å hente FNR dersom vi mottar en aktørId, og har ellers ryddet bort noe logikk som tar hensyn til aktørid
Lagt til PdlService

olsnerik committed:
YSMOD-637
- Konvertere aktørId til fødselsnummer i Dokumentmottak
- PdlService som et abstraksjonslag rundt PdlClient, erstatter bruk av PdlClient med PdlService
- Justert tester og testdata

olsnerik committed:
YSMOD-637
- Fjerne kommentarer fra koden
- Tilpasninger, nytt test case i DokumentmottakTest

olsnerik committed:
YSMOD-637 Fikset test

olsnerik committed:
YSMOD-637
- Legge til PdlClientTest
- Fikse package path for enkelte graphql fixtures

lurajon reviewed (approved):
Bra å få PDL ut til egen service.

Vi kunne vurdert å brukt faker for å generere FNR i diverse mockito returns.

olsnerik committed:
YSMOD-637
- Sende maskintilmaskin-token til PDL ved mottak av nytt dokument
- ny enum Tokentype lar oss bestemme tokentype i klientene
- renamet clientcredentials -> maskintilmaskin for PDL token
- Flyttet mocking fra klargjorDatabase() til setup()
- Andre tilpasninger i tester
